### DISCUSSION TITLE
How to go back to index file

### DISCUSSION THREAD

**Discussion by KlapenHz** [nvim-neorg/neorg#1134] [Q&A] [answered]
Hi, 
I'm sorry if the question has already been asked, 
I have index file (after putting: `:Neorg index`
with link to another file for example:
`{:PATH/neorg.norg:}[Neorg]`
When I press enter and move to this file, how to get back to index file without putting again `:Neorg index`?
Thank you for any answers

**Comment by pysan3** [comment, marked as answer]
Do you mean `<C-o>`?

**Reply by KlapenHz** [reply]
Thanks a lot, it works. I don't have this shortcut in: 
https://github.com/nvim-neorg/neorg/blob/main/lua/neorg/modules/core/keybinds/keybinds.lua
Is it described somewhere else?

**Reply by Anrock** [reply]
It's default vim shortcut that goes to previous position before jump

**Reply by KlapenHz** [reply]
Ah, ok, I never use vim to work with many files in the same time. Ok, good to know. Thanks.

**Reply by pysan3** [reply]
FYI, jumping to the previous file is `<C-^>`. `<C-6>` works the same for most cases.

**Reply by KlapenHz** [reply]
Thanks for help :)

**Reply by cjoke** [reply]
This is late, but if you in the document you created link to you can do {:index:} <-- this will point back to index.norg file.

**Comment by shaw4m4n** [comment]
Use `<C-o>` and `<C-i>` to navigate back and forth through location history.
